docs/gds-supported-platform/helm-chart-best-practices.md

```diff
@@ -0,0 +1,92 @@
+# Helm Chart Best Practices
+
+## Overview
+
+Guidance for authoring Helm chart's suitable for production deployments.
+
+### Use the release name in resource names
+
+`.metadata.name` should include at least `{{ .Release.Name }}`
+
+For example:
+
+```
+metadata:
+    name: {{ .Release.Name }}-my-app
+```
+
+This allows you to install multiple versions of your application to a single
+namespace without clashing/clobbering resources with the same name.
+
+### High Availability
+
+Replicas should be specified per application, rather than globaly so that they can we scaled individualy:
```

Comment:
```suggestion
Replicas should be specified per application, rather than globally so that they can we scaled individually:
```
also, some note about taking into account limitations given by application's state, such as rate limiting